roboteam_ai/src/coach/OffensiveCoach.cpp

```diff
@@ -65,12 +67,14 @@ std::vector<Vector2> OffensiveCoach::getDefaultLocations() {
     std::vector<Vector2> defaultPositions;
 
     // Calculate two positions close to goal
```

Comment:
the function is called : getDefaultLocations. Weird name. what does it do? Could be better.

Reply:
changed in to zoneLocations, and added a comment